src/inverseRobot.js

```diff
@@ -8,6 +8,27 @@
 
 function inverseRobot(robot) {
   // write code here
+  let tempKyesRobot = {};
+  let tempValueRobot = {};
+
+  tempKyesRobot = Object.keys(robot);
+  tempValueRobot = Object.values(robot);
```

Comment:
Similarly, there is no need to initialize `tempValueRobot` with an empty object on line 12 since you are assigning it a new value here. You can declare and assign it in one line.